lib/cloudstore/rate_limiter.es6.js

```diff
@@ -0,0 +1,62 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
```

Comment:
This file doesn't seem specific to the cloud datastore implementation. It probably shouldn't be in this directory.